resources/service-catalog-addons/charts/service-catalog-ui/templates/deployment.yaml

```diff
@@ -19,13 +19,19 @@ spec:
     spec:
       containers:
         - name: {{ .Chart.Name }}
-          image: "{{ .Values.global.containerRegistry.path }}{{ .Values.image.dir }}/brokers-ui:{{ .Chart.AppVersion }}"
+          image: "{{ .Values.global.containerRegistry.path }}{{ .Values.image.dir }}/service-catalog-ui:{{ .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           ports:
-            - containerPort: {{ .Values.service.internalPort }}
+            - containerPort: {{ .Values.catalog.service.internalPort }}
+            - containerPort: {{ .Values.instances.service.internalPort }}
+            - containerPort: {{ .Values.brokers.service.internalPort }}
           volumeMounts:
           - name: config
-            mountPath: /var/public/config
+            mountPath: /var/catalog-public/config
```

Comment:
Maybe also mount nginx config from config-map like in add-ons-ui? https://github.com/kyma-project/kyma/blob/master/resources/helm-broker/charts/add-ons-ui/templates/deployment.yaml#L35